setup.py

```diff
@@ -22,7 +22,7 @@
     "install_requires": required,
     "extras_require": {
         "spark": ["pyspark>=2.3.2"],
-        "sqlalchemy": ["sqlalchemy>=1.2,<1.4.0"],
+        "sqlalchemy": ["sqlalchemy>=1.2"],
```

Comment:
@Shinnnyshinshin : How come `requirements-dev-sqlalchemy.txt` specifies version `>=1.3.16` while `setup.py` version `>=1.2` -- should they not be the same for consistency?  Thanks!

Reply:
great point :) Thanks for catching this